src/test/java/uk/gov/hmcts/reform/professionalapi/service/impl/OrganisationServiceImplTest.java

```diff
@@ -237,7 +237,6 @@ public void retrieve_an_organisations_by_status() {
         OrganisationsDetailResponse organisationDetailResponse = sut.findByOrganisationStatus(OrganisationStatus.ACTIVE);
     }
 
-
```

Comment:
Not essential, but if no real changes made in this class then revert it

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-professional-api/pull/612/commits/acbe4881fa2e5d46be921bc81e0a93e8f8bf9f4f